.github/workflows/build-emscripten.yml

```diff
@@ -59,3 +59,18 @@ jobs:
     - name: Build
       if: success()
       uses: DiligentGraphics/github-action/build@v1
+
+    - name: Pack artifacts
+      if:  ${{ (success() || failure() && steps.build.outcome == 'success') && (matrix.host_name == 'Linux')  && (matrix.build_type == 'Release') }}
```

Comment:
We only need to upload artifacts if all steps succeed.
The `failure() && steps.build.outcome == 'success'` condition is needed for Samples to upload golden images in case some test fails so that new golden images can be downloaded and examined.
Here we don't need this.